bmc/firmware.go

```diff
@@ -2,201 +2,163 @@ package bmc
 
 import (
 	"context"
-	"errors"
 	"fmt"
 	"io"
 
-	"github.com/hashicorp/go-multierror"
-)
+	bmclibErrs "github.com/bmc-toolbox/bmclib/errors"
 
-// BMCVersionGetter retrieves the current BMC firmware version information
-type BMCVersionGetter interface {
-	GetBMCVersion(ctx context.Context) (version string, err error)
-}
+	"github.com/pkg/errors"
 
```

Comment:
nit: maybe keeping github.com imports in a single block will help the diff be more readable?